src/pages/yc-onboarding.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import Layout from '../components/Layout'
+import { DemoScheduler } from '../components/DemoScheduler'
+import { Spacer } from '../components/Spacer'
+import { Link } from 'gatsby'
+import { Button } from 'antd'
+import './styles/yc-onboarding.scss'
+
+const DemoCallInfo = () => (
+    <>
+        <p>
+            Welcome to our super secret YC onboarding page! Given that YC now runs batches remotely for the time being,
+            we decided to set up this page in order to provide a simple way to schedule demos that suits startups across
+            all timezones. If you simply can't find a time that suits you (maybe you're in Perth?), then feel free to
+            email us at <i>yc@posthog.com</i> and we'll be happy to find a time that works.
+        </p>
+        <br />
+        <p>
+            The demo call will take place with Yakko, one of our developers, and runs for about an hour. Before the
```

Comment:
I'd call it a "setup call" to be clearer

Reply:
Will update!